### DISCUSSION TITLE
Laravel Homestead Unable to load FFProbe

### DISCUSSION THREAD

**Discussion by csteamengine** [spatie/laravel-medialibrary#2312] [Q&A]
Hello, I have been using your project and I love it, but recently I was trying to setup video conversions to display a thumbnail of an uploaded video. I have run `composer require php-ffmpeg/php-ffmpeg` and have installed both `ffmpeg` and `ffprobe` via Homebrew, but when I try to upload a video through my Homestead site, I get this error:
```
FFMpeg\Exception\ExecutableNotFoundException
Unable to load FFProbe
```
I have ensured that my `.env` file is setup correctly with the paths to `ffmpeg` and `ffprobe` respectively:
```
# MEDIA LIBRARY
FFMPEG_PATH=/usr/local/bin/ffmpeg
FFPROBE_PATH=/usr/local/bin/ffprobe
```
and I have ensured that the laravel-medialibrary is getting the correct paths by injecting a `dump()` statement right before this exception is thrown, but have still been unable to get ffprobe working. 
``` 
//In FFProbeDriver.php
        try {
            return static::load($binaries, $logger, $configuration);
        } catch (BinaryDriverExecutableNotFound $e) {
            throw new ExecutableNotFoundException('Unable to load FFProbe', $e->getCode(), $e);
        }
```

Any help you could provide would be greatly appreciated, I'd love to get video conversions working!

Extra Info:
* MacOs Catalina (2015 Macbook Pro)
* Laravel Framework: ^8.19
* I'm using a Homestead environment, downloaded within the past couple weeks
* FFProbe: 4.3.2
* FFMPeg: 4.3.2

**Comment by redtangle-uk** [comment]
@csteamengine Did you ever resolve this, it looks similar to a problem i'm having.
